k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1210,3 +1297,26 @@ if [[ ${num_motr_client} -gt 0 ]]; then
     deployCortxClient
 fi
 cleanup
+
+
+# TODO: get service names from the script above rather than hard-coding it
+data_service_name="cortx-io-svc-0"
+data_service_default_user="sgiamadmin"
```

Comment:
Pull these fields from the solution.yaml definitions like the TODO says instead of hard-coding it.

Reply:
Will do.

Reply:
@osowski , @keithpine , I need some advice.  I can get `data_service_default_user` from solution.yaml, but the others are more tricky, in part because of this bash/helm hybrid.

* `data_service_name` is present in `cortx-platform/values.html`...sort of.  `cortx-io-svc` is the value in `values.yaml`, and the `-0` is tagged on as part of the `cortx-io-svc.yaml` template.
* `control_service_name` is an input parameter to `helm install "cortx-control...`  I could set a variable for that, but it feels a little weird/out of the normal flow.  I could also remove this `--set` parameter from helm install and just set it in `values.yaml`.  This doesn't really fix anything, but it would make it consistent with these other problem cases.
* `control_service_default_user` is set in `cortx-configmap/templates/_config.tpl`

So for two of these three, this deploy script does not have access to these fields.  I thought adding this "helpful" information would be nice for users.  I admit it's not great to hardcode the values, but I don't see an good alternative.

What do you advise?

Reply:
After verbal discussion we agreed that I would leave these hard-coded values as-is for this PR.  I will open an issue with information on what needs to be changed to address this from top to bottom and put it into the backlog.